{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/common/views.py

```diff
@@ -18,7 +19,7 @@ def index(request):
 
 def index(request):
     try:
-        return render(request, "index.html")
+        return render(request, "index.html", status=status.HTTP_404_NOT_FOUND)
     except TemplateDoesNotExist:
-        return render(request, "core/index-placeholder.html")
+        return render(request, "core/index-placeholder.html", status=status.HTTP_404_NOT_FOUND)
```

Comment:
You'll still get the index placeholder returned, but that status for that return will now be a `404` instead of a `200`.
This is particularly useful when writing some tests. If your URL was bad (ex: extra `/`) and you don't have a response object to inspect, your test would pass since you got a 200 on a URL route that doesn't exist.